packages/common/src/operations.generated.ts

```diff
@@ -4,17 +4,6 @@ import { GraphQLClient } from 'graphql-request';
 import * as Dom from 'graphql-request/dist/types.dom';
```

Comment:
this is a big diff for this file! I just took *out* the `itemListView` query and put it in the `system/item` package operations file.

There is another item related query here: `itemsWithStockLines` which I *duplicated* into the `system/item` package as it is also used in `system/stock`. These queries would likely differ in the future so I think duplicating the query in *each* package is likely the way to go.